auditlog/management/commands/auditlogflush.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
+
+from auditlog.models import LogEntry
+
+
+class Command(BaseCommand):
+    help = "Deletes all log entries from the database."
+
+    def add_arguments(self, parser):
+        parser.add_argument('-y, --yes', action='store_true', default=None,
+                            help="Continue without asking confirmation.", dest='yes')
+
+    def handle(self, *args, **options):
+        answer = options['yes']
+
+        while answer is None:
+            print("This action will clear all log entries from the database.")
+            response = input("Are you sure you want to continue? [y/N]: ").lower().strip()
+            answer = True if response == 'y' else False if response == 'n' else None
```

Comment:
Functionally it is different. In case there is no valid response, the user is prompted again for an answer.

Could write it out longer, maybe in a separate function, for readability.

Reply:
Ah, yes, with the `while`. 

Though, I wonder if the 'N' should be lower cased? I've understood the upper case letter to typically mean that will be the response if one simply hits "enter"/"return"

Reply:
Good point, will address that too.